lib/active_admin_sidebar/activeadmin_views_pages_base.rb

```diff
@@ -24,6 +32,8 @@ def main_content_classes
     else
       classes << 'with_sidebar'
       classes << 'left_sidebar' if left_sidebar?
+      classes << 'collapsible_sidebar' if collapsible_sidebar? # collapsible option is activated
+      classes << 'collapsed_sidebar' if collapsible_sidebar? && sidebar_is_collapsed? # sidebar is collapsed
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.